.github/workflows/test-e2e-samples.yml

```diff
@@ -43,8 +43,8 @@ jobs:
           sed -i '25s/^#//' $KUSTOMIZATION_FILE_PATH
           sed -i '47,49s/^#//' $KUSTOMIZATION_FILE_PATH
           # Uncomment all cert-manager injections
-          sed -i '59,212s/^#//' $KUSTOMIZATION_FILE_PATH
-          sed -i '214,229s/^#//' $KUSTOMIZATION_FILE_PATH
+          sed -i '59,234s/^#//' $KUSTOMIZATION_FILE_PATH
+          sed -i '236,251s/^#//' $KUSTOMIZATION_FILE_PATH

```

Comment:
Hi @abhishekdwivedi3060 ^ that was why the e2e tests were failing

Reply:
Hi @camilamacedo86, I was not aware of this required change.
I ran `make test-e2e-local` and it passed, so couldn't catch this.

Reply:
This has been tested in GitHub Actions. However, I'm not sure what happens when a PR gets merged and the tests are skipped.

It's a really weird issue because once it was merged, we started encountering problems on the master branch as well. See: [GitHub Actions Run](https://github.com/kubernetes-sigs/kubebuilder/actions/runs/13218789559).